java/ql/src/Likely Bugs/Concurrency/DoubleCheckedLockingWithInitRace.ql

```diff
@@ -0,0 +1,43 @@
+/**
+ * @name Race condition in double-checked locking object initialization
+ * @description Performing additional initialization on an object after
+ *              assignment to a shared variable guarded by double-checked
+ *              locking is not thread-safe, and could result in unexpected
+ *              behavior.
+ * @kind problem
+ * @problem.severity error
```

Comment:
Perhaps classify this query more conservatively as a `warning`, since its results tend not to be quite as readily identifiable as definite `error`s as those of the other query.

Reply:
fixed